_includes/components/hero-sm.njk

```diff
@@ -1,9 +1,9 @@
 <div class="hero hero-sm">
 	<div class="hero-img lazy"></div>
 	<div class="hero-title">
-		<h1 class="hero-title-text"><span class="mono">GET</span> <span class="audiowide">CONF</span> <span class="hero-title-date">4/26/19</span></h1>
+		<h1 class="hero-title-text"><span class="mono">GET</span> <span class="audiowide">CONF</span> <span class="hero-title-date">Gender Equity in Tech</span></h1>
```

Comment:
I decided to replace the date of the conference with our name.

Reply:
Great catch! I (finally) updated our Twitter profile this weekend, too.